src/test/java/com/openlattice/conductor/ConductorBootstrap.java

```diff
@@ -31,29 +31,26 @@
  * @author Matthew Tamayo-Rios &lt;matthew@openlattice.com&gt;
  */
 public class ConductorBootstrap {
+
     protected static final Indexer INDEXER;
 
     static {
-        INDEXER = new Indexer();
         final var logger = LoggerFactory.getLogger( ConductorBootstrap.class );
+        INDEXER = new Indexer();
+        String[] profiles = new String[3];
+        profiles[0] = "postgres";
+        profiles[1] = "keras";
```

Comment:
I don't think we need keras in Indexer.. I think it's just a leftover from when linker was part of indexer.
Locally I use this btw: `local postgres medialocal auditlocal`

Reply:
Good call, I was being too mechanical with this 🤦‍♂ 
Will update